source/api_reference/search_payments_reference/index.html.md.erb

```diff
@@ -186,7 +186,7 @@ After you send your request to search payments, you’ll receive a response simi
 | `settlement_summary.capture_submit_time` | date (ISO 8601) | The date and time GOV.UK Pay asked your payment service provider to take the payment from your user's account.<br><br>This value uses Coordinated Universal Time (UTC) and ISO 8601 format - `YYYY-MM-DDThh:mm:ss.sssZ`.<br><br>`capture_submit_time` only appears if GOV.UK Pay has asked your PSP to take the payment. |
 | `settlement_summary.captured_date` | date (ISO 8601) | The date your payment service provider took the payment from your user. <br><br>This value appears in ISO 8601 format - `YYYY-MM-DD`.<br><br>`captured_date` only appears if your payment service provider has taken the payment. |
 | `settlement_summary.settled_date` | date (ISO 8601) | The date your payment service provider sent the payment to your bank account as part of a payout. <br><br>This value appears in ISO 8601 format - `YYYY-MM-DD`.<br><br>`settled_date` only appears if your payment service provider is Stripe and they have sent the payment to your bank account. |
-| `card_details` | object | Contains details about the card the user paid with.<br><br>`card_details` only appears if the paying user has completed this payment. |
+| `card_details` | object | Contains details about the card the user paid with.<br><br>Some details in `card_details` may not be accurate if the user paid with [a digital wallet](/digital_wallets/#restrictions-of-digital-wallet-payments).<br><br>`card_details` only appears if the paying user has completed this payment. |
```

Comment:
I’m going to challenge the term “accurate” here. It’s not that the information is wrong, it’s just that it may not match the details on the physical card in some cases.

Reply:
@alexbishop1 That's a really good point. 'Accurate' makes it sound a bit sloppy. What do you think about?:

> Some information in `card_details` may not match your user's physical card. This is because of [how Apple Pay and Google Pay handle cardholder information](LINK).

Reply:
Yeah, that’s better but maybe it should be a single sentence to make it clearer the limitation only applies to digital wallets?

Reply:
@alexbishop1 

> If your user paid with a digital wallet, some information in `card_details` may not match your user's physical card. This is because of [how Apple Pay and Google Pay handle cardholder information](LINK).